### PR TITLE
archtype updates

### PR DESCRIPTION
**Pull request by isao**
- enable -p option for `create mojit`
- update app/hybrid mojito controller to 0.5.x

### CODE REVIEW

File: lib/app/archetypes/app/hybrid/mojits/top_frame/controller.server.js.hb [hunk 1]
```diff
@@ -43,4 +42,9 @@ YUI.add('top_frame', function(Y, NAME) {
 
     };
 
-}, '0.0.1', {requires: ['mojito', 'top_frameModelFoo']});
+}, '0.0.1', {requires: [
+    'mojito',
+    'mojito-assets-addon',
+    'mojito-models-addon',
+    'top_frameModelFoo'
```
Comment (mojit0) [inline lib/app/archetypes/app/hybrid/mojits/top_frame/controller.server.js.hb:49]: I think this has to change to something ala '{{name}}ModelFoo'.

**Comment by mojit0** [comment]
One comment, otherwise +1

**Comment by drewfish** [comment]
+1

File: lib/app/archetypes/app/hybrid/mojits/top_frame/controller.server.js.hb (hunk 1 above)
Comment (drewfish) [inline lib/app/archetypes/app/hybrid/mojits/top_frame/controller.server.js.hb:49]: This is an app archetype, which contains a canned "top_frame" mojit, so I think what Isao has is correct.

Comment (isao) [inline lib/app/archetypes/app/hybrid/mojits/top_frame/controller.server.js.hb:49]: it's inconsistent, I know. the hybrid archetype is supposed to include both an app and a mojit and the mojit name was hard-coded (was a requirement)

**Pull request merged**